README.md

```diff
@@ -36,6 +36,21 @@ This bundle can be installed following these steps:
         resource: @IntegratedPageBundle/Resources/config/routing.xml
         prefix: "/admin"
 
+## Using contentType pages ##
+
+In order for contentType pages to work you need to define your controllers as services (examples can be found in Resources/config/controllers.xml).
+
+In the service you then have to tag your controller as a contentType controller by adding the tag "integrated_page". Furthermore the "class" attribute is required in this tag.
+By default the showAction will be called for the contentType controller, but you can also define one or more actions in the "controller_actions" attribute.
+
+some examples:
+
+    <tag name="integrated_page.contenttype_controller" class="Integrated\Bundle\ContentBundle\Document\Content\Article"/>
+    <tag name="integrated_page.contenttype_controller" class="Integrated\Bundle\ContentBundle\Document\Content\Relation\Company" controller_actions="fooAction, showAction"/>
```

Comment:
Dont use comma seperators. Why not use something like this:

`
<tag name="integrated_page.contenttype_controller" class="Integrated\Bundle\ContentBundle\Document\Content\Relation\Company">
<controller-action>fooAction</action>
<controller-action>showAction</action>
</tag>
`